drodrpg/DRODLib/DbRooms.cpp

```diff
@@ -4974,42 +4974,40 @@ const
 	return this->pszTParams[ARRAYINDEX(wX,wY)];
 }
 
+//*****************************************************************************
+void CDbRoom::SnapCoordsOutsideOfRoomToEdge(int& nX, int& nY) const
```

Comment:
Clamp would probably be a better word to use, since snap suggests snapping to grid. Maybe `ClampCoordsToRoom`?

Reply:
That's a good callout.  The room is a grid, but I agree Clamp makes more sense with the logic of the function bounding outside coords to the room edges.  I'll make the change.